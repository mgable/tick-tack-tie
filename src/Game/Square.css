.square {
	display: flex;
	justify-content: center;
	align-items: center;
	border: 1px solid black;
	width: 50px;
	height: 50px;
	margin-right: -1px;
	margin-top: -1px;
	font-size: 36px;
}

.square:focus {
	outline: 0 none;
}