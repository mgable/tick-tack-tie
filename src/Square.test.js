import React from 'react';
import ReactDOM from 'react-dom';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

// import App from './App';
import Square from './Game/components/Square'

// const mockStore = configureStore();
// const initialState = {}

// const store = mockStore(initialState);

describe('App ', () => {
	test('renders without crashing', () => {
		const wrapper = shallow(<Square value={1} onClick={(idx) => idx}/>)
		// const component = wrapper.dive();
		expect(toJson(wrapper)).toMatchSnapshot();
	})
	
});
