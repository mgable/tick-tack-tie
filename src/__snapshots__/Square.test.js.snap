// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App  renders without crashing 1`] = `
<button
  className="square"
  onClick={[Function]}
>
  1
</button>
`;
